corehq/apps/userreports/tasks.py

```diff
@@ -130,7 +130,7 @@ def rebuild_indicators(indicator_config_id, initiated_by=None, limit=-1, source=
     timeout=3 * 60 * 60, max_retries=20,
     queue=UCR_CELERY_QUEUE, ignore_result=True
 )
-def rebuild_indicators_in_place(indicator_config_id, initiated_by=None, source=None):
+def rebuild_indicators_in_place(indicator_config_id, initiated_by=None, source=None, domain=None):
```

Comment:
since `domain` is unused in the function it would useful to add a comment indicating why it's there otherwise it's likely to get removed at some later point.